src/cache_dit/serve/serve.py

```diff
@@ -22,7 +22,24 @@ def get_args(
     parser.add_argument("--compile", action="store_true", default=False)
     parser.add_argument("--compile-repeated-blocks", action="store_true", default=False)
     parser.add_argument("--max-autotune", action="store_true", default=False)
-    parser.add_argument("--fuse-lora", action="store_true", default=False)
+    parser.add_argument(
+        "--lora-path", type=str, default=None, help="Path to LoRA weights directory"
+    )
+    parser.add_argument(
+        "--lora-name", type=str, default=None, help="LoRA weight filename (e.g., model.safetensors)"
+    )
+    parser.add_argument(
+        "--fuse-lora",
+        action="store_true",
+        default=True,
+        help="Fuse LoRA weights into model (recommended for better performance)",
+    )
+    parser.add_argument(
```

Comment:
The flag '--disable-fuse-lora' seems redundant. It should be sufficient to keep either '--fuse-lora' or '--disable-fuse-lora'. What do you think?

Reply:
done.